src/libstore/build.cc

```diff
@@ -59,10 +59,12 @@
 /* chroot-like behavior from Apple's sandbox */
 #if __APPLE__
     #define SANDBOX_ENABLED 1
-    #define DEFAULT_ALLOWED_IMPURE_PREFIXES "/System/Library /usr/lib /dev /bin/sh"
+    #define DEFAULT_ALLOWED_IMPURE_PREFIXES "/System/Library /usr /dev /bin/sh"
```

Comment:
We still need to set *some* __impureHostdeps for top-level dependencies like `/usr/lib/libSystem.dylib`.